pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dev_dependencies:
   args: ^2.0.0
   grinder: ^0.9.0-nullsafety.0
   path: ^1.8.0
+  path_provider: ^2.0.0-nullsafety.1
```

Comment:
I was expecting this to be a dependency override. Why is it a dev_dependency?

Reply:
Updated this to an override